django/core/management/commands/migrate.py

```diff
@@ -54,6 +55,10 @@ def add_arguments(self, parser):
             '--run-syncdb', action='store_true',
             help='Creates tables for apps without migrations.',
         )
+        parser.add_argument(
+            '--plan', action='store_true', dest='plan',
```

Comment:
`dest='plan',` is unneeded as that's the default value from `--plan` (eac9ab7ebb1ce0cbbc79c4cf65e8f70b0635a240)

Reply:
Will fix